tests/test_config.py

```diff
@@ -145,3 +145,19 @@ def test_ranges(self):
         conf = Conf()
         with self.assertRaises(dnf.exceptions.ConfigError):
             conf.debuglevel = '11'
+
+    def test_releasever_major_minor(self):
+        conf = Conf()
+        conf.releasever = '1.2'
+        self.assertEqual(conf.releasever_major, '1')
+        self.assertEqual(conf.releasever_minor, '2')
+
+        # override releasever_major
+        conf.releasever_major = '3'
+        self.assertEqual(conf.releasever_major, '3')
+        self.assertEqual(conf.releasever_minor, '2')
+
+        # override releasever_minor
+        conf.releasever_minor = '4'
+        self.assertEqual(conf.releasever_major, '3')
+        self.assertEqual(conf.releasever_minor, '4')
```

Comment:
I would also assert that `conf.releasever == '1.2'` if that is the requested behavior.

Reply:
Done.